### ISSUE TITLE
Info: Potentially bad value sitemap for attribute rel on element link

### ISSUE THREAD

**Issue by edent** [validator/validator#2081]
### Summary

When using `<link rel=sitemap …>` the checker raises a warning.

### URL for document that reproduces the problem

https://shkspr.mobi/blog/

### HTML/CSS/SVG snippet that reproduces the problem

```html
<link rel=sitemap href=https://example.com/sitemap.xml type=application/xml>
```

### Checker error messages

```shell
Info: Potentially bad value sitemap for attribute rel on element link Typo for timemap?
```

### Spec links/citations

https://html.spec.whatwg.org/multipage/links.html#other-link-types

That says:

> Extensions to the predefined set of link types may be registered on the [microformats page for existing rel values](https://microformats.org/wiki/existing-rel-values#HTML5_link_type_extensions). [[MFREL]](https://html.spec.whatwg.org/multipage/references.html#refsMFREL)

There is a rel="sitemap" registered on that site - https://microformats.org/wiki/rel-sitemap

While I don't have hard statistics, it is used in some popular CMS software, and appears to be widely used in open source projects:

* https://jekyllcodex.org/without-plugin/sitemap/
* https://docs.astro.build/en/guides/integrations-guide/sitemap/
* https://package.elm-lang.org/packages/canceraiddev/elm-pages/latest/Head
* https://github.com/search?q=+rel%3D%22sitemap%22&type=code

### Steps to reproduce

1. Check a page with the rel="sitemap" attribute.
2. See error.

### Expected behavior

Checker should be aware of the popular rels registered on microformats.

### Actual behavior

An error occurs.

**Comment by j9t** [comment]
> There is a rel="sitemap" registered on that site - https://microformats.org/wiki/rel-sitemap

That is marked as “proposed” though. Just pointing it out—aware of this process but not clear about how and when a value is considered official and “standard.”

**Comment by edent** [comment]
Indeed, but so are things like `amphtml` and `apple-touch-icon`. Those are all valid in:

https://github.com/validator/validator/blob/b51a3e066c8f4bd9b2e63517603117b7487eee8b/src/nu/validator/datatype/RelValue.java#L38-L52

I assume the validator only uses the rel values from https://www.iana.org/assignments/link-relations/link-relations.xhtml

So I'll raise an issue at https://github.com/protocol-registries/link-relations/
